packages/components/src/public/QueryModel/GridPanel.tsx

```diff
@@ -64,6 +66,7 @@ type Props<T> = GridPanelProps<T> & RequiresModelAndActions;
 
 interface GridBarProps<T> extends Props<T> {
     onViewSelect: (viewName) => void;
+    onAliquotViewChange: (filter: Filter.IFilter, filterColumnToRemove?: string) => void;
```

Comment:
I know this is possibly only a temporary solution, but since ButtonBar and GridPanel are meant to be very general-purpose and are in the "public" domain, we probably want to call this something else.  Maybe `onFilteredViewChange`?  Of course, the filtering component itself is aliquot-specific as well...  Can we generalize?

Reply:
We could generalize the GridAliquotViewSelector to a FilteredViewSelector (works with boolean column only) but. I prefer to punt that when we do the work to merge it with grid view selector.